app/models/user_avatar.rb

```diff
@@ -152,6 +152,13 @@ def self.import_url_for_user(avatar_url, user, options = nil)
   end
 
   def self.ensure_consistency!(max_optimized_avatars_to_remove: 20_000)
+    DB.exec <<~SQL
+      DELETE FROM user_avatars
+      USING user_avatars ua
+      LEFT JOIN users u ON ua.user_id = u.id
+      WHERE user_avatars.id = ua.id AND u.id IS NULL
+    SQL
```

Comment:
This query scales better compared to:
```
  DELETE FROM user_avatars
  WHERE user_id NOT IN (
    SELECT id FROM users
  )
```
When running EXPLAIN ANALYZE on a database with ~1.5 million users, the LEFT JOIN query showed execution time of `746.723 ms` vs `921.336 ms`.